_data/projects.yml

```diff
@@ -63,19 +63,21 @@ project metadata as frequently as every few minutes. This growing flow of update
 has created a large and potentially devastating new attack surface for malicious
 actors, as malware submitted through this process offers the opportunity to
 impact thousands of users at a time. The Update Framework (TUF) is a comprehensive,
-flexible approach to securing software updates, even in situations where the repository
+flexible framework to secure software updates, even in situations where the repository
 is compromised. Developers can integrate TUF into any software update system, or
 native programming language due to its concise, self-contained architecture and
-specification. It’s adoption in 2017 by the <a href=\"https://www.cncf.io\">Cloud Native Computing Foundation</a>
+application. It’s adoption in 2017 by the <a href=\"https://www.cncf.io\">Cloud Native Computing Foundation</a>
 of the <a href=\"https://www.linuxfoundation.org/\">Linux Foundation</a> along
 with Docker's popular implementation of the framework, now makes TUF the de facto
 security standard for the cloud."
 
-      products: "<a href=\"https://theupdateframework.com\">TUF</a> is used in
-      production by a <a href=\"https://theupdateframework.github.io/adoptions.html\">variety of
-projects</a>, including <a href=\"https://www.vmware.com/\">VMware</a>,
-<a href=\"https://blog.cloudflare.com/pal-a-container-identity-bootstrapping-tool/\">Cloudflare</a>, and
-<a href=\"https://blog.docker.com/2015/08/content-trust-docker-1-8/\">Docker</a>.
+      products: "<a href=\"https://theupdateframework.com\">TUF</a> has been adopted by a growing number of
+companies, including <a href=\"https://cloud.ibm.com/docs/services/Registry?topic=registry-registry_trustedcontent#registry_trustedcontent\">IBM Cloud Container Registry</a>, 
+<a href=\"https://docs.microsoft.com/en-us/azure/container-registry/container-registry-content-trust\">Microsoft Azure</a>, 
+<a href=\"https://www.vmware.com/\">VMware</a>, 
+<a href=\"https://www.airbiquity.com/news/press-releases/airbiquity-unveils-new-otamatictm-release-further-strengthening-over-air-ota-software-and-data-management-offering-automotive\">Airbiquity OTAmatic</a>,
+<a href=\"https://blog.cloudflare.com/pal-a-container-identity-bootstrapping-tool/\">Cloudflare</a>,
+<a href=\"https://blog.docker.com/2015/08/content-trust-docker-1-8/\">Docker</a>. and <a href=\"https://www.here.com/products/automotive/ota-technology\">Here Technologies</a>.
 It has been standardized for Python as documented in PEPs <a href=\"https://www.python.org/dev/peps/pep-0458/\">458</a> and
 <a href=\"https://www.python.org/dev/peps/pep-0480/\">480</a>. And, through CNCF,
 TUF even has <a href=\"https://store.cncf.io/collections/tuf\">merch</a>" for sale.
```

Comment:
I fixed other issues per your request, including replacing the original CacheCash text. I also made a few additional edits on the retired projects, mostly for Sensibility and Seattle. It felt weird sending people to a website that was now dated. So I added copy that affirms that the material posted is archival and no longer being updated.